packages/select/src/components/multi-select/multiSelect.tsx

```diff
@@ -33,12 +34,18 @@ import {
     type TagInputProps,
     Utils,
 } from "@blueprintjs/core";
-import { Cross } from "@blueprintjs/icons";
+import { Cross, Search } from "@blueprintjs/icons";
 
 import { Classes, type ListItemsProps, type SelectPopoverProps } from "../../common";
 import { QueryList, type QueryListRendererProps } from "../query-list/queryList";
 
 export interface MultiSelectProps<T> extends ListItemsProps<T>, SelectPopoverProps {
+    /**
+     * Element which triggers the multiselect popover. Providing this prop will replace the default TagInput
```

Comment:
could you read through the full docs page? pretty much anything referencing `TagInput` may need to be updated now